jsapp/js/components/submissions/table.es6

```diff
@@ -1044,6 +1043,94 @@ export class DataTable extends React.Component {
 
     return mediaURL;
   }
+  cellDisplaysNumbers(questionOrKey) {
+    let questionType = questionOrKey;
+    if (questionOrKey.type) {
+      questionType = questionOrKey.type;
+    }
+
+    return (
+      NUMERICAL_SUBMISSION_PROPS[questionType] ||
+      TABLE_MEDIA_TYPES.includes(questionType)
+    );
+  }
+  launchMediaModal(questionType, questionIcon, mediaURL, mediaName) {
+    stores.pageState.showModal({
+      type: MODAL_TYPES.TABLE_MEDIA_PREVIEW,
+      questionType: questionType,
+      mediaURL: mediaURL,
+      mediaName: mediaName,
+      customModalHeader: this.renderMediaModalCustomHeader(
+        questionIcon,
+        mediaURL,
+        mediaName
+      ),
+    });
+  }
+  renderMediaModalCustomHeader(questionIcon, mediaURL, mediaName) {
+    return (
+      <bem.TableMediaPreviewHeader>
+        <bem.TableMediaPreviewHeader__title>
+          <i className={questionIcon.join(' ')}/>
+          <bem.TableMediaPreviewHeader__label>
+            {mediaName}
+          </bem.TableMediaPreviewHeader__label>
+        </bem.TableMediaPreviewHeader__title>
+
+        {/*TODO this doesn't start a `save as` but instead opens media in tab*/}
+        <a
+          className='kobo-light-button kobo-light-button--blue'
+          href={mediaURL}
+          download=''
+        >
+          {t('download')}
+          <i className='k-icon k-icon-download'/>
+        </a>
+      </bem.TableMediaPreviewHeader>
+    );
+  }
+  renderMediaCell(questionType, mediaURL, mediaName) {
```

Comment:
Please make a `mediaCell.es6` file with this simple code, and move styles for it to `mediaCell.scss`. Normally I wouldn't mind this, but `table.es6` (and `.scss`) is too huge and we should try moving things out if possible